provisioners/roles/newrelic_wsgi/tasks/main.yml

```diff
@@ -64,5 +64,6 @@
 - name: import newrelic to wsgi
   lineinfile: dest={{wsgi_scriptfile}} state=present line="import newrelic.agent" insertbefore=BOF
   sudo: yes
+  ignore_errors: yes
```

Comment:
you need to ignore errors in the tasks task as well as the handler task? Since it's only failing in the handler, I'm surprised that's necessary.